site/_resources/compatibility-matrix.md

```diff
@@ -104,6 +104,7 @@ __Note:__ This list of extensions was last verified to be complete with Envoy v1
 [10]: https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.17.1
 [11]: https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.17.2
 [12]: https://www.envoyproxy.io/docs/envoy/latest/version_history/v1.18.2
+[13]: https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current
```

Comment:
huh didnt realize they changed the site like this

Reply:
I think the release notes should make it into the `/latest` site eventually, but they're not there yet